docs/reference/elements/py-config.md

```diff
@@ -75,7 +75,8 @@ One can also use both i.e pass the config from `src` attribute as well as specif
 
 ```html
 <py-config src="./custom.toml">
-  paths = ["./utils.py"]
+  [[fetch]]
+  files = ["./utils.py"]
 </py-config>
```

Comment:
[suggestion] This example has the purpose of showing that you can use both `src` and inline content, it's probably better to use an easier-to-understand config settings, e.g. `packages` or `name`.

Reply:
done, using `packages`